docs/04-how-we-work/continuous-learning.md

```diff
@@ -0,0 +1,54 @@
+
+CivicActions is a learning organization, stemming from our organizational belief in the continuous learning
+principles of Agile. All work that each of us performs presents opportunities for continuous learning. We aim 
+to learn from one another and share our knowledge back to make the world better.
+
+## Project Learning
+
+Projects are defined in broad context to capture any sort of work performed within the team. This could be a 
+development project for a client, writing a sales proposal, a marketing initiative, or even professional development
+activities.
+
+Our Scrum teams hold retrospectives to make continuous improvements based on what is learned during projects.
+Projects can iteratively make changes that improve efficiency, operations, and much more. This approach encourages 
+team members to learn, grow, and experiment in ways to more effectively serve our customers without fear of 
+failure. More information is provided on the [retrospectives](agile-baseline/02-process/practices/sprint-retrospectives.md) 
+page. 
+
+## Organizational Learning
+
+As we work, retrospectives may identify considerations or improvements outside of projects that generally affect how 
+we work. Team members may organically come up with ideas that they want to share with CivicActions. We aim to provide 
+a framework for harnessing and discussing their ideas within the organization.
+
+We have split feedback into groups to afford more focused discussions. Some groups are leveraging Holacracy and Agile 
+Guild concepts as a framework for organizational learning. This is akin to providing an interface for our team members 
+to share thoughts with the broader organization. More details can be found on the 
+[Holacracy](/docs/04-how-we-work/holacracy.md) page. 
+
+Each group maintains their own process for soliciting feedback, which is described below. Considerations to form a group 
+can be found [here](/docs/04-how-we-work/holacracy.md).
+
+### Departments
+
+All continuous learning practices are maintained department-by-department. Many of the departments are leveraging 
+Holacracy and Guild concepts to collect tensions, review during a governance meeting, and define outcomes.
+
+More information can be found on each department from the [department listing](../02-about-us/departments.md).
+
+### Services
+
+Each of our [services](../02-about-us/services) could be candidates for their own guilds. But, as a small team 
+with limited resources, we may not have the broad participation desired for guilds. 
+
+As an alternative, services could be affiliated with departmental governance meetings. Tensions can be raised that 
+may relate to a specific services. Collaboration can happen during the meeting and outcomes determined, which may include 
+follow up actions or further discussion pertinent to the service.
+
+### Working Groups
+
+CivicActions can create [working groups](../02-about-us/working-groups) that pertain to initiatives or efforts as the 
```

Comment:
Working groups are different than guilds. Working groups form from initiatives (e.g. labs, documentation, etc). Guilds have periodic meetings, discuss improvements, etc. One is more around defining a lightweight process for reviewing feedback, the other is a structure for moving an initiative forward